src/shared/components/results/ResultsContainer.tsx

```diff
@@ -84,10 +84,11 @@ const Results: FC = () => {
   const initialApiUrl = getAPIQueryUrl({
     namespace,
     query,
-    columns,
+    columns: namespace === Namespace.uniparc ? undefined : columns,
```

Comment:
Not too keen on that but it's linked to the hack L81 (instead of using `[]` we should explicitly define the columns we want to retrieve for cards and for table for each namespace, and I guess "undefined" would be accepted). Maybe move this to L81 so all the hack is in the same place and add a comment here https://www.ebi.ac.uk/panda/jira/browse/TRM-25288

Reply:
I can't really do that in l.81 as the value as it is in l.81 is required in l.180 and passed down to the result view component in order to render the correct columns. l.87 is the only place where I need to override it

Reply:
Ok just add to the Jira 👍